### PR TITLE
Integrated React & made a simple Hello World page

### LINKED ISSUE #3: Create a simple hello world with react

### LINKED ISSUE #9: Repository setup 🚀
- [ ] [Update README.md with project description, getting started, prerequisites, installation and setup instructions.](https://github.com/girijaasoni/newsletterApplicaton/issues/7)
- [x] [Initialize the app with Ruby on Rails 7, PostgreSQL and get a server up and running.](https://github.com/girijaasoni/newsletterApplicaton/issues/8)
- [ ] ~~Secure db credentials~~

### PR DESCRIPTION
**Pull request by girijaasoni**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2,6 +2,956 @@
 # yarn lockfile v1
 
 
+"@ampproject/remapping@^2.1.0":
```
Comment (Ron-Lavi) [inline yarn.lock]: Can you add this file to .gitignore and remove it?

File: config/database.yml [hunk 2]
```diff
@@ -58,6 +58,10 @@ development:
 test:
   <<: *default
   database: newsletterApplicaton_test
+  host: localhost
+  port: 5432
+  username: rail_user
+  password: Redhat!123
```
Comment (Ron-Lavi) [inline config/database.yml]: I think we need to keep the credentials hidden, can you also remove this file and add it to .gitignore?

In foreman we created the same file with .example in the end of file name so devs could reuse it as a template.

**Review by Ron-Lavi — COMMENTED** [review]
Nice work @girijaasoni !!

File: .gitignore [hunk 3]
```diff
@@ -34,3 +34,5 @@
 !/app/assets/builds/.keep
 
 /node_modules
+/yarn.lock
+/config/database.yml
```
Comment (Ron-Lavi) [inline .gitignore:38]: Can you drop the `/` from this and the above lines? The files weren't ignored

**Comment by Ron-Lavi** [comment]
@archanaserver could you also review and test if everything works?
After your approval I could merge it

File: config/database.yml (hunk 2 above)
Comment (archanaserver) [reply config/database.yml]: @Ron-Lavi can we do in this way, with `EDITOR=nano rails credentials:edit` , update username and password here. Then update the database.yml with:

```
username: <%= Rails.application.credentials.database[:username] %>
password: <%= Rails.application.credentials.database[:password] %>
```

Reference: https://levelup.gitconnected.com/setting-up-postgres-with-ruby-on-rails-7-on-ubuntu-20-04-45ffd75e863d

File: README.md [hunk 4]
```diff
@@ -1,24 +1,4 @@
...
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+To run the application, 
+run "./bin/dev" and in the terminal they mention the port link, 
+use that link to run the application on localhost
```
Comment (archanaserver) [inline README.md:4]: That's a great start for us! But I think let's add a few more details to make it more readable for others, by adding all of these: https://github.com/girijaasoni/newsletterApplicaton/issues/7 (I'm on it!)

**Review by archanaserver — COMMENTED** [review]
Dropped a few suggestions.

File: config/database.yml (hunk 2 above)
Comment (Ron-Lavi) [reply config/database.yml]: Actually not a bad idea, let's try it maybe in a follow up, for now I suggested using it as in Foreman

**Comment by archanaserver** [comment]
> @archanaserver could you also review and test if everything works? After your approval I could merge it

@Ron-Lavi I have tested it locally and it is working fine.

**Review by Ron-Lavi — APPROVED** [review]
Looks good to me, great work @girijaasoni and @archanaserver :)

**Pull request merged**